functions/tests/integration/test_location_with_firestore.py

```
@@ -0,0 +1,346 @@
+        # 6. Return
+
+        # For now, test with local data since Firestore integration
+        # requires emulator setup
+        result = await get_location_factors("80202")
+
+        assert isinstance(result, LocationFactors)
+        assert result.zip_code == "80202"
+        assert len(result.labor_rates) == 8
+
+    @pytest.mark.asyncio
+    async def test_firestore_document_structure_matches_schema(self, firestore_client):
```

Ruff (0.14.8)

Unused method argument: `firestore_client`

(ARG002)